internal/store/node.go

```diff
@@ -530,3 +532,12 @@ func createNodeListWatch(kubeClient clientset.Interface, _ string, _ string) cac
 		},
 	}
 }
+
+const maxValue = math.MaxInt64 / 1000
```

Comment:
You might want to compare against this one here:
https://github.com/kubernetes/kubernetes/blob/3f5d0ee2cf402d10233fc403750a9784d1542059/staging/src/k8s.io/apimachinery/pkg/api/resource/math.go#L49

Reply:
thanks, I have already fixed it.